.github/actions/cache-path/action.yml

```diff
@@ -0,0 +1,53 @@
+name: Cache path on host using github actions cache
+
+description: Add CHANNEL to GITHUB_ENV
+inputs:
+  cache-path:
+    description: "If Caching is used provide valid path"
```

Comment:
Why caching is capitalized? and valid path to what? I.e. `/tmp` is a valid path, isn't it?

Reply:
Done, added proper description.

Reply:
Provided description for path.